build.gradle.kts

```diff
@@ -779,7 +779,9 @@ tasks {
         !it.name.startsWith("client") && !it.name.startsWith("filesystem") && !it.name.startsWith("spark") && !it.name.startsWith("iceberg") && it.name != "trino-connector" &&
         it.name != "integration-test" && it.name != "bundled-catalog" && !it.name.startsWith("flink") &&
         it.name != "integration-test" && it.name != "hive-metastore-common" && !it.name.startsWith("flink") &&
-        it.name != "gcp-bundle" && it.name != "aliyun-bundle" && it.name != "aws-bundle" && it.name != "azure-bundle" && it.name != "hadoop-common"
+        it.name != "gcp-bundle" && it.name != "aliyun-bundle" && it.name != "aws-bundle" && it.name != "azure-bundle" &&
+        it.name != "aws-hadoop-bundle" && it.name != "gcp-hadoop-bundle" && it.name != "azure-hadoop-bundle" && it.name != "aliyun-hadoop-bundle" &&
```

Comment:
In theory, all modules under bundles do not need to copy dependencies.

 Can we optimize the judgment condition here so that we can add sub-modules under bundles later without modifying this judgment condition?

Reply:
Use `endsWith("bundle")` or `contains("bundle")`?

Reply:
how about `(it.parent?.name) != "bundles"`

Reply:
Okay, I will take it.